src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1307,7 +1307,7 @@ This version used in: {2}</value>
   <data name="Switching_between_lambda_and_local_function_will_prevent_the_debug_session_from_continuing" xml:space="preserve">
     <value>Switching between a lambda and a local function will prevent the debug session from continuing.</value>
   </data>
-  <data name="Switching_between_lambda_and_local_function_will_prevent_the_debug_session_from_continuing" xml:space="preserve">
```

Comment:
Was this removed intentionally? Seems unrelated...

Reply:
It was duplicate of one above.

---
In reply to: [138757442](https://github.com/dotnet/roslyn/pull/21980#discussion_r138757442) [](ancestors = 138757442)